scripts/kind-e2e/e2e.sh

```diff
@@ -311,25 +323,41 @@ if [[ $1 != keep ]]; then
     trap cleanup EXIT
 fi
 
-echo Starting with status: $1, k8s_version: $2, logging: $3, kubefed: $4.
+echo Starting with status: $1, k8s_version: $2, logging: $3, kubefed: $4, broker :$5.
```

Comment:
This will be like "make ci e2e .... broker=cluster2" for example right. This should be mentioned somewhere. Maybe in testing.md from my doc PR #176

Reply:
Yes, it should indeed! Once #176 is merged we’ll have to make it a requirement that user-visible behaviour-changing patches also update the documentation. (And this patch mustn’t go in before #176; in any case it needs substantial rework following the merge of the operator patch.)